pkg/infrastructure/persistence/mappers.go

```diff
@@ -35,7 +35,8 @@ func toDomainUser(dbUser *models.User) *user.User {
 		MiddleName: middleName,
 		Email:      dbUser.Email,
 		Password:   password,
-		//AvatarID:   dbUser.AvatarID,
+		Avatar:     toDomainUpload(dbUser.Avatar),
```

Comment:
I would rather move the nil check here. toDomainUpload should seg fault when handling a nil pointer